src/main/java/com/jvm_bloggers/frontend/public_area/blogs/BlogPostsPage.java

```diff
@@ -0,0 +1,81 @@
+package com.jvm_bloggers.frontend.public_area.blogs;
+
+import com.jvm_bloggers.entities.blog.BlogRepository;
+import com.jvm_bloggers.entities.blog_post.BlogPost;
+import com.jvm_bloggers.entities.blog_post.BlogPostRepository;
+import com.jvm_bloggers.frontend.admin_area.PaginationConfiguration;
+import com.jvm_bloggers.frontend.common_components.infinite_scroll.InfinitePaginationPanel;
+import com.jvm_bloggers.frontend.common_components.request_handlers.BlogPostsPageRequestHandler;
+import com.jvm_bloggers.frontend.public_area.AbstractFrontendPage;
+
+import org.apache.wicket.markup.html.WebMarkupContainer;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.link.ExternalLink;
+import org.apache.wicket.markup.repeater.Item;
+import org.apache.wicket.markup.repeater.data.DataView;
+import org.apache.wicket.request.mapper.parameter.PageParameters;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.wicketstuff.annotation.mount.MountPath;
+
+import static com.jvm_bloggers.frontend.public_area.blogs.BlogPostsPage.BLOG_ID_PARAM;
+import static com.jvm_bloggers.utils.DateTimeUtilities.DATE_FORMATTER;
+
+@MountPath("blogs/${" + BLOG_ID_PARAM + "}/posts")
```

Comment:
My latest idea was about using `author` field with `domain` ending then we could have nice url fragments:
```
lukasz-kuczynski-io
konrad-ktoso-malawski-ski
slawek-sobotka-com
```

Generate this fragment from `domain` will force us to support all different cases. For instance:
```
http://usethiscode.blogspot.com/ (blogspot)
https://solidsoft.wordpress.com (wordpress)
https://mateuszrasinski.github.io/ (github)
https://blog.pchudzik.com/index.xml (custom with blog)
http://blog.wittchen.biz.pl (custom with blog)
https://medium.com/@jaceklaskowski?source=rss-df186e1cd0c2------2 (medium)
https://touk.pl/blog (custom with blog)
https://www.youtube.com/channel/UC2coGyxf5x_CzJ3l4F-N-Sw 
https://vimeo.com/geecon/videos
```